### PR TITLE
Add Volume.CubicHectometer

### PR DESCRIPTION
**Pull request by cpmoliveira**
Add Length.Hecto prefix
Add Volume.CubicHectometer
Add test HectometersInOneMeter
Add test CubicHectometersInOneCubicMeter

Modify the test Assert.Equal with addition of InvariantInfo, because when you compare string in cultures like Brazilian Portuguese (the decimal separator is comma) the test fail.

### CODE REVIEW

**Comment by angularsen** [comment]
Looks good to me, just need to fix merge conflicts. Basically just regenerate code and push.

File: UnitsNet.WindowsRuntimeComponent/UnitsNet.WindowsRuntimeComponent.csproj [hunk 1]
```diff
@@ -11,21 +11,19 @@
     <AssemblyName>UnitsNet</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.16299.0</TargetPlatformVersion>
+    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.17763.0</TargetPlatformVersion>
```
Comment (angularsen) [inline UnitsNet.WindowsRuntimeComponent/UnitsNet.WindowsRuntimeComponent.csproj]: I assume this change was not intentional? Possibly by just opening the project in visual studio. If so, just revert this file.

Comment (cpmoliveira) [reply UnitsNet.WindowsRuntimeComponent/UnitsNet.WindowsRuntimeComponent.csproj]: Yes, it was not intentional.
Thank you for the patience.

**Comment by angularsen** [comment]
Nuget on the way out.
[Release UnitsNet/4.14.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F4.14.0)

**Pull request merged**